components/__tests__/BlurhashCanvas.spec.ts

```diff
@@ -14,45 +14,35 @@ beforeEach(() => {
 
 describe('BlurhashCanvas', () => {
   test('has a hash property of type String', () => {
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-    expect(wrapper.vm.$options.props.hash).toBeDefined();
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-    expect(wrapper.vm.$options.props.hash.type).toBe(String);
+    // @ts-expect-error - props are not defined in the Vue type.
+    expect(wrapper?.vm?.$options?.props?.hash).toBeDefined();
+    // @ts-expect-error - props are not defined in the Vue type.
+    expect(wrapper?.vm?.$options?.props?.hash?.type).toBe(String);
   });
 
-  test('has a width property of type Number', () => {
```

Comment:
I don't think you meant to delete this line

Reply:
Oops, good catch!